moto/core/responses.py

```diff
@@ -18,6 +18,7 @@
 from typing import Dict, List, Union, Any, Optional
 from urllib.parse import parse_qs, parse_qsl, urlparse
 from werkzeug.exceptions import HTTPException
+import moto.prettify_jinja_render  # noqa # pylint: disable=unused-import
```

Comment:
Sorry, I wasn't very clear earlier. I was hoping to have all the logic here, instead of in a separate file.
It is better to extend the existing method that stores the XML, IMO. In pseudocode:
```
def response_template(self, source):
    template_id = self._make_template_id(source)
        if not self.contains_template(template_id):
            if PRETTIFY:
                pretty_xml = parseXML(..)
            else:
                collapsed = re.sub(
                    self.RIGHT_PATTERN, ">", re.sub(self.LEFT_PATTERN, "<", source)
                )
            self.environment.loader.update({template_id: collapsed or pretty_xml})
        return self.environment.get_template(template_id)
```

 - easier to reason if the logic is in one place
 - it doesn't clutter the root folder
 - it prevents people from treating this behaviour as a public API, by calling `from moto import prettify_jinja_render` in their own code